spk/syncthing/src/dsm-control.sh

```diff
@@ -3,68 +3,94 @@
 # Package
 PACKAGE="syncthing"
 DNAME="Syncthing"
+DINAME="Syncthing-Inotify"
 
 # Others
 INSTALL_DIR="/usr/local/${PACKAGE}"
 PATH="${INSTALL_DIR}/bin:${PATH}"
 USER="syncthing"
+CONFIG_DIR="${INSTALL_DIR}/var"
+# Syncthing
 SYNCTHING="${INSTALL_DIR}/bin/syncthing"
-CONFIG_DIR="${INSTALL_DIR}/var/"
-
+SYNCTHING_OPTIONS="-home=${CONFIG_DIR}"
+# Syncthing-Inotify
+SYNCTHING_INOTIFY="${INSTALL_DIR}/bin/syncthing-inotify"
+SYNCTHING_INOTIFY_OPTIONS="-home=${CONFIG_DIR}"
+# Include user configurable options
+. ${CONFIG_DIR}/syncthing-addoptions.cfg
 
 start_daemon ()
 {
-    start-stop-daemon -b -o -c ${USER} -S -u ${USER} -x env HOME=${CONFIG_DIR} ${SYNCTHING} -- --home ${CONFIG_DIR}
+    start-stop-daemon -b -o -c ${USER} -S -u ${USER} -x env HOME=${CONFIG_DIR} $1 -- $2
 }
```

Comment:
yes, I agree that the script looks weird now. I tried to keep original variable names. I would rather rework it completely (get rid of 'SYNCTHING' in variables) to make it easy adaptable to run any two binaries simultaneously (e.g. 'DAEMON1' and 'DAEMON2').